unit-requirements.txt

```diff
@@ -2,4 +2,5 @@ pytest
 pytest-mock
 coverage[toml]
 requests
+ipdb
```

Comment:
Do we need this?

Reply:
We set `PYTHONBREAKPOINT=ipdb.set_trace`, shouldn't we install `ipdb` to enable setting breakpoints?

Reply:
I hadn't seen this line, so maybe you do!